RELEASE_NOTES.md

```diff
@@ -1,21 +1,20 @@
-# Cosmos SDK v0.50.2 Release Notes
+# Cosmos SDK v0.50.3 Release Notes
 
 💬 [**Release Discussion**](https://github.com/orgs/cosmos/discussions/58)
 
 ## 🚀 Highlights
 
-A month ago, Cosmos SDK Eden was released. Missed the announcement? Read it [here](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.1).
-For second patch release of the v0.50.x line, a few features and improvements were added to the SDK.
+Some months ago Cosmos SDK Eden was released. Missed the announcement? Read it [here](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.50.1).
+For this month patch release of the v0.50.x line, a few features and improvements were added to the SDK.
 
 Notably, we added and fixed the following:
 
-* Allow to import base64 encoded pubkeys in the keyring using `<appd> keys add <name> --pubkey-base64 <base64-pubkey>` 
-* A bug when migrating from v0.45/v0.46 directly to v0.50 due to missing `ConsensusParams` 
-* An issue when simulating gas for transactions when using a multisig
+* 
+*
```

Comment:
nit: these are empty

Reply:
yes, this is why there is "do not merge", I want to bump some deps here too.

Reply:
shoot dude, sorry I completely missed that.